airflow/hooks/druid_hook.py

```diff
@@ -83,6 +84,12 @@ def construct_ingest_query(
         else:
             num_shards = -1
 
+        if not query_granularity:
```

Comment:
Nit: query_granularity = query_granularity or "NONE" is more pythonic (and same for below).